docker/docker-compose.yml

```diff
@@ -5,62 +5,62 @@ services:
   postgres:
     image: postgres:9.6
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      test: ["CMD-SHELL", "pg_isready -U root" ]
     restart: always
     ports:
-    - 5432:5432
+      - 5432:5432
```

Comment:
Please try to keep unrelated whitespace changes in their own separate commit. It makes it hard to understand code reviews, and breaks `git annotate`.

Reply:
I have a habit of reformatting. We should have a shared intelij format. I've added a IDEAS ticket to look at it.

Reply:
I'm not saying don't reformat; I'm merely saying be disciplined when doing so. Use `git add -p` to ensure that your commits only do one thing at a time.